lib/swig/array/ssparsearray_typemap_in.i

```diff
@@ -88,6 +87,10 @@
 // The final macro for dealing with sparse arrays
 %define SSPARSEARRAY_TYPEMAPIN_MACROS(SSPARSEARRAY_TYPE, SSPARSEARRAYPTR_TYPE, SSPARSEARRAY2D_TYPE, SSPARSEARRAY2DPTR_TYPE,
                                       SPARSEARRAY_TYPE, SPARSEARRAY2D_TYPE, C_TYPE)
+
 TYPEMAPIN_SSPARSEARRAY(SSPARSEARRAY_TYPE, SSPARSEARRAYPTR_TYPE, SPARSEARRAY_TYPE, SPARSEARRAY2D_TYPE, C_TYPE);
-TYPEMAPIN_SSPARSEARRAY2D(SSPARSEARRAY2D_TYPE, SSPARSEARRAY2DPTR_TYPE, SPARSEARRAY_TYPE, SPARSEARRAY2D_TYPE, C_TYPE)
+TYPEMAPIN_SSPARSEARRAY2D(SSPARSEARRAY2D_TYPE, SSPARSEARRAY2DPTR_TYPE, SPARSEARRAY_TYPE, SPARSEARRAY2D_TYPE, C_TYPE);
+//#error SColMaj##SPARSEARRAY2D_TYPE##
```

Comment:
This //error comment is frightening

Reply:
good catch - was testing if the file was even used